compiler/qsc_rca/tests/common/mod.rs

```diff
@@ -0,0 +1,160 @@
+        self.compute_properties = analyzer.analyze_package(package_id);
+    }
+}
+
+impl Default for CompilationContext {
+    fn default() -> Self {
+        let compiler = Compiler::new(
+            true,
+            SourceMap::default(),
+            PackageType::Lib,
+            RuntimeCapabilityFlags::all(),
+            LanguageFeatures::default(),
+        )
+        .expect("should be able to create a new compiler");
+        let mut lowerer = Lowerer::new();
+        let fir_store = lower_hir_package_store(&mut lowerer, compiler.package_store());
+        let analyzer = Analyzer::init(&fir_store);
+        let compute_properties = analyzer.analyze_all();
+        Self {
+            compiler,
+            fir_store,
+            lowerer,
+            compute_properties,
+        }
+    }
+}
+
+pub trait PackageStoreSearch {
+    fn find_callable_id_by_name(&self, name: &str) -> Option<StoreItemId>;
+}
+
+impl PackageStoreSearch for PackageStore {
+    fn find_callable_id_by_name(&self, name: &str) -> Option<StoreItemId> {
+        for (package_id, package) in self.iter() {
+            if let Some(item_id) = package.find_callable_id_by_name(name) {
+                return Some((package_id, item_id).into());
+            }
+        }
+
+        None
+    }
+}
+
+pub trait PackageSearch {
+    fn find_callable_id_by_name(&self, name: &str) -> Option<LocalItemId>;
+}
+
+impl PackageSearch for Package {
+    fn find_callable_id_by_name(&self, name: &str) -> Option<LocalItemId> {
+        for (item_id, item) in &self.items {
+            if let ItemKind::Callable(callable_decl) = &item.kind {
+                if callable_decl.name.name.as_ref() == name {
+                    return Some(item_id);
+                }
+            }
+        }
+
+        None
+    }
+}
+
+pub fn check_callable_compute_properties(
+    fir_package_store: &impl PackageStoreSearch,
+    package_store_compute_properties: &PackageStoreComputeProperties,
+    callable_name: &str,
+    expect: &Expect,
+) {
+    let callable_id = fir_package_store
+        .find_callable_id_by_name(callable_name)
+        .expect("callable should exist");
+
+    let callable_compute_properties = package_store_compute_properties.get_item(callable_id);
+    expect.assert_eq(&callable_compute_properties.to_string());
+}
+
+pub fn check_last_statement_compute_properties(
+    package_store_compute_properties: &PackageStoreComputeProperties,
+    expect: &Expect,
+) {
+    let last_package_id = package_store_compute_properties
+        .iter()
+        .map(|(package_id, _)| package_id)
+        .sorted()
+        .last()
+        .expect("at least one package should exist");
+    let package_compute_properties = package_store_compute_properties.get(last_package_id);
+    let last_statement_id = package_compute_properties
+        .stmts
+        .iter()
+        .map(|(stmt_id, _)| stmt_id)
+        .sorted()
+        .last()
+        .expect("at least one statement should exist");
```

Comment:
The same thing can be done here:

```suggestion
    let last_statement_id = package_compute_properties
        .stmts
        .iter()
        .map(|(stmt_id, _)| stmt_id)
        .max()
        .expect("at least one statement should exist");
```